cmd/orchard-status/main.go

```
@@ -0,0 +1,98 @@
+		if s.Pid <= 0 || syscall.Kill(s.Pid, 0) != nil {
+			continue
+		}
+		live = append(live, s)
+	}
+	sort.Slice(live, func(i, j int) bool { return live[i].Ts > live[j].Ts })
+
+	if asJSON {
+		if live == nil {
+			live = []sessFile{}
+		}
+		json.NewEncoder(os.Stdout).Encode(live)
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)